packages/next/src/server/app-render/after-task-async-storage-instance.ts

```diff
@@ -0,0 +1,5 @@
+import type { AfterTaskAsyncStorage } from './after-task-async-storage.external'
+import { createAsyncLocalStorage } from './async-local-storage'
+
+export const _afterTaskAsyncStorage: AfterTaskAsyncStorage =
```

Comment:
The `_` prefix seems weird

Reply:
the intention here is to not accidentally auto-import `afterTaskAsyncStorage` from this file, because we always want to import it from `.external`. i guess i've done that one times too many

Reply:
renamed to `afterTaskAsyncStorageInstance`